### PR TITLE
Add precisions about DEV plan on custom extensions

### PR DESCRIPTION
**Pull request by Bencaddyro**
## Describe your PR

Add a warning box on PG custom extension section

## Checklist

- [x] My PR is related to an opened issue : #279
- [x] I've read the [contributing guidelines](/CleverCloud/documentation/blob/main/CONTRIBUTING.md)

## Reviewers
_Who should review these changes?_ @juliamrch

### CODE REVIEW

File: content/doc/addons/postgresql/_index.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ pgtap       | Unit testing for PostgreSQL
 pgvector    | Vector data type and ivfflat and hnsw access methods
 timescaledb | Enables scalable inserts and complex queries for time-series data (Community Edition)
 
+{{< callout type="warning" >}}
+On-demand extensions are not available for DEV addons.
```
Comment (judu) [inline content/doc/addons/postgresql/_index.md:131]:
```suggestion
On-demand extensions are not available for DEV plans.
```

Comment (judu) [reply content/doc/addons/postgresql/_index.md:131]: Also, maybe use "aren't" like the bot said? :shrug:

**Review by judu — COMMENTED** [review]
Yes, I'm nitpicking on a single line.

File: content/doc/addons/postgresql/_index.md (hunk 1 above)
Comment (juliamrch) [reply content/doc/addons/postgresql/_index.md:131]: Yeah, we unified the term. It's `add-on` and it flags `addon`

```suggestion
On-demand extensions aren't available for DEV plans.
```

**Review by juliamrch — CHANGES_REQUESTED** [review]

**Comment by juliamrch** [comment]
@Bencaddyro feel free to commit our suggestions from linter or to change it yourself and we're good to go

**Comment by Bencaddyro** [comment]
@juliamrch suggestions applied

**Review by juliamrch — APPROVED** [review]

**Pull request merged**